webapp/src/views/History.tsx

```
@@ -0,0 +1,415 @@
+      <div className="max-w-7xl mx-auto space-y-6">
+        {/* Header */}
+        <div className="flex items-center justify-between">
+          <div className="flex items-center gap-3">
+            <HistoryIcon className="w-8 h-8 text-primary" />
+            <h1 className="text-3xl font-bold">Measurement History</h1>
+          </div>
+          <button
+            onClick={exportToCSV}
+            disabled={!measurements || measurements.length === 0}
+            className="flex items-center gap-2 px-4 py-2 bg-primary text-primary-foreground rounded-md hover:bg-primary/90 disabled:opacity-50 disabled:cursor-not-allowed"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)